contracts/migrations/Migratable.sol

```diff
@@ -50,7 +50,8 @@ contract Migratable {
    * @param newMigrationId Identifier of the new migration to be applied.
    */
   modifier isMigration(string contractName, string requiredMigrationId, string newMigrationId) {
-    require(isMigrated(contractName, requiredMigrationId) && !isMigrated(contractName, newMigrationId));
+    require(isMigrated(contractName, requiredMigrationId), "Requested base migration ID has not been run yet");
```

Comment:
The concept of "base migration" was never established. How about something like "_Prerequisite_ migration has not been run yet"?